src/platform/site-wide/sass/merger.scss

```diff
@@ -557,9 +559,17 @@
             margin: 2px;
           }
 
+          &:focus:hover {
+            background-color: $mm-gray;
+            font-weight: bold;
+            margin: 2px;
+            width: 98%;
+          }
+
```

Comment:
Is `&:focus:hover` definitely what we want to be doing here?

Reply:
Let me try to add the 100% to the parent and see if this fixes things.